packages/contracts-bedrock/scripts/deploy/DeployImplementations.s.sol

```diff
@@ -124,6 +128,7 @@ contract DeployImplementations is Script {
             protocolVersionsImpl: address(_output.protocolVersionsImpl),
             l1ERC721BridgeImpl: address(_output.l1ERC721BridgeImpl),
             optimismPortalImpl: address(_output.optimismPortalImpl),
+            optimismPortalInteropImpl: address(_output.optimismPortalInteropImpl),
```

Comment:
Should we run post-deployment assertions for the optimismPortalInterop output. Similar to the checks for the regular portal - https://github.com/ethereum-optimism/optimism/blob/3aab5f96d66e228f63386bdbb7c86f00a3f9b79a/packages/contracts-bedrock/scripts/deploy/DeployImplementations.s.sol#L669.

Reply:
Will do this in a different PR, it's not as trivial as it seems